src/components/modules/tools.ts

```diff
@@ -472,3 +473,22 @@ export default class Tools extends Module {
     };
   }
 }
+
+/**
+ * What kind of plugins developers can create
+ */
+export enum toolTypes {
+  /**
+   * Block tool
+   */
+  BLOCK,
```

Comment:
Enum values also usually go in PascalCase
```suggestion
  Block,
```